wolfcrypt/src/port/aria/aria-crypt.c

```diff
@@ -0,0 +1,275 @@
+/* aria.c
+ *
+ * Copyright (C) 2006-2023 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+/*
+
+DESCRIPTION
+This library provides the interfaces to the ARIA cipher, an encryption algorithm
+developed by the Korean Agency for Technology (KATS). It uses a 128-bit block
+size and a key size of 128, 192, or 256 bits.
+
+*/
+#ifdef HAVE_CONFIG_H
+    #include <config.h>
+#endif
+
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/wolfcrypt/error-crypt.h>
+#include <wolfssl/ssl.h>
+#include <wolfssl/wolfcrypt/port/aria/aria-crypt.h>
+
+#ifdef HAVE_ARIA
+    /* return 0 on success or WC_INIT_E on failure */
+    int wc_AriaInitCrypt(wc_Aria* aria, MC_ALGID algo)
+    {
+        MC_RV rv = MC_OK;
+
+        MC_APIMODE gApimode = MC_MODE_KCMV;
+        MC_ALGMODE algMode = MC_ALGMODE_GCM;
+        MC_PADTYPE algPad = MC_PADTYPE_NONE;
+
+        if (aria == NULL) return BAD_FUNC_ARG;
```

Comment:
Please put if () and statement on separate lines throughout. No if() statement on same line.